src/ansys/dpf/core/misc.py

```diff
@@ -80,9 +80,7 @@ def get_ansys_path(ansys_path=None):
     # First check the environment variable for a custom path
     if ansys_path is None:
         ansys_path = os.environ.get("ANSYS_DPF_PATH")
-    # Then check for usual installation folders with AWP_ROOT and find_ansys
-    if ansys_path is None:
-        ansys_path = os.environ.get("AWP_ROOT" + __ansys_version__)
+    # Then check among installed packages, AWP_ROOT variables, and usual installation folders
     if ansys_path is None:
         ansys_path = find_ansys()
```

Comment:
@PProfizi is `find_ansys()` doing ` ansys_path = os.environ.get("AWP_ROOT" + __ansys_version__)`?

Reply:
@cbellot000, no, as this was the actual issue.
If you prioritize `AWP_ROOT+__ansys_version__`, then you do not use the standalone installed in your venv.

With the new implementation, `find_ansys` goes through all installed `ansys-dpf-server` and takes the latest if any, then goes through `AWP_ROOT` variables and takes the latest if any, which results in the same thing as prioritizing `__ansys_version` which is the latest anyway. This is actually one thing less to maintain. Or then I am missing a use-case